storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.16.rst

```diff
@@ -0,0 +1,35 @@
+================================================================================
+|Percona XtraBackup| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona XtraBackup| |release| on
+|date|. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.16/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories. 
+
+All Percona software is open-source and free.
+
+Improvements
+================================================================================
+
+- Add timeout options for FTWRL, LTFB, LBFB. Bug fixed :jirabug:`PXB-1914`
+
+Bugs Fixed
+================================================================================
+
+- |Percona Xtrabackup| was not able to connect to the database when
+  the password was specified along with the transition-key
+  parameter. Bug fixed :jirabug:`PXB-1902`.
+- In some cases, |Percona Xtrabackup| stuck with redo log corruption
+  when master key is rotated. Bug fixed :jirabug:`PXB-1903`.
+- |Percona Xtrabackup| could crash during the incremental backup
+  prepare for the tablespace encryption. Bug fixed
```

Comment:
Please clarify that PXB could crash during incremental backup prepare only in case when both full and incremental backups were made before MySQL flushed first page of encrypted tablespace (in other words almost never in real world)